app/models/form_attachment.rb

```diff
@@ -34,6 +34,24 @@ def get_file
 
   private
 
+  def unlock_pdf(file, file_password)
+    return file unless File.extname(file) == '.pdf'
+
+    pdftk = PdfForms.new(Settings.binaries.pdftk)
+    tmpf = Tempfile.new('decrypted')
```

Comment:
I'm not sure if it is matters but do you need to specify the extension for the temp file here with Tempfile.new(['decrypted', '.pdf'])?